src/javascript/JContent/ContentNavigation/ContentNavigation.jsx

```diff
@@ -6,7 +6,9 @@ import {useTranslation} from 'react-i18next';
 const ContentNavigation = ({accordionItems, mode, siteKey, handleNavigation, header}) => {
     const {t} = useTranslation('jcontent');
     return (
-        <SecondaryNav header={header}>
+        <SecondaryNav defaultSize={{height: '100%', width: '245px'}}
```

Comment:
is defaultSize  supported somehow ?  anyway 100% height is not really possible as the parent container has no fixed height